helpers/.component-template/README.md

```diff
@@ -1,3 +1,7 @@
-# @westpac/**COMPONENT_KEY**
+Note: Remove below prettier-ignore line
```

Comment:
Is there a nicer way of handling this?

Reply:
Did Thinkmill use the ** syntax because it's valid markdown? Can your replace function just check for ** and __ wrapped strings?

Reply:
Prettier autoformats __ to **. Im just going to refactor everything to use a single underscore